src/catalogue/category/__snapshots__/cataloguePropertyForm.component.test.tsx.snap

```diff
@@ -2267,8 +2267,8 @@ exports[`Catalogue Property Form > renders correctly when disabled 1`] = `
         <label
           class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary Mui-disabled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
           data-shrink="false"
-          for=":ri:"
-          id=":ri:-label"
+          for="undefined-propety-unit-input"
+          id="undefined-propety-unit-input-label"
```

Comment:
I had to try to find a way to make each id unique, when there is multiple properties in the form. Otherwise for some reason the labels all duplicate onto only the first textfield. I didn't realise `cip_placement_id` could be undefined, I'll find a fix for it